README.md

```diff
@@ -1,3 +1,24 @@
+# Algorithm names
+
+The first round of "winners" of the [NIST PQC competition](https://csrc.nist.gov/Projects/post-quantum-cryptography) are Dilithium, Falcon, SPHINCS+, and Kyber. As of August 23, 2023 with the publication of the Initial Public Drafts of FIPS 203, 204, and 205 we know that Dilithium is being renamed to "ML-DSA" (Module Lattice), SPHINCS+ to "SLH-DSA" (StateLess Hash), and Kyber to "ML-KEM (Module Lattice).
+
+However, the community wants to avoid the situation where we over-eagerly adopt the new names, then see a compatibility-breaking change in a future iteration of the FIPS draft standards, and then have multiple non-compatible algorithms with the same name. 
+
+The IETF will use the following naming convention to refer to different versions of the NIST PQC candidate algorithms; using Dilithium -> ML-DSA as an example:
+
+* _**Dilithium_round1/2/3**_ -- means “as submitted to round 1, 2, or 3 of the NIST PQC competition”. “Dilithium” is a short-hand for this.
+
+* _**ML-DSA-ipd**_ – FIPS 204 Initial Public Draft. Note this matches the PDF file name:
+https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.204.ipd.pdf
+ 
+* _**ML-DSA-pfYYMM**_ -- reserved for additional rounds of changes to the FIPS 204 draft standard; "pfYYMM" stands for "pre-final" and the year and month of publication.
+
+* _**ML-DSA**_ – reserved for final FIPS 204.
```

Comment:
It would be nice for the final names to not have years / month in them, we've had a lot of discussions similar to this in the W3C on names containing version or time information, and that causing secondary effects like:

"This is too old, we need a new one"... or "Now's my chance to make a new version, even if the old one is still fine to use"...

I don't have a perfect answer here, but I think we should be cautious about how naming shapes incentives.

Reply:
I propose dropping the pfYYMM line altogether. If we get into that situation, we can add a new name there based on what NIST calls it.

Reply:
yes fine, let's drop the date information.

Reply:
I will accept this, but I can also believe that people will want to remove it because it is not really about "state of protocols"